docs/src/ensemble_kalman_inversion.md

```diff
@@ -3,24 +3,28 @@
 One of the ensemble Kalman processes implemented in `EnsembleKalmanProcesses.jl` is the ensemble
 Kalman inversion ([Iglesias et al, 2013](http://dx.doi.org/10.1088/0266-5611/29/4/045001)).
 The ensemble Kalman inversion (EKI) is a derivative-free ensemble optimization method that seeks
-to find the optimal parameters ``\theta \in \mathbb{R}^p`` in the inverse problem
+to find the optimal parameters ``\theta \in \mathbb{R}^p`` in the inverse problem defined by the data-model relation
 
 ```math
-y = \mathcal{G}(\theta) + \eta ,
+\tag{1} y = \mathcal{G}(\theta) + \eta ,
 ```
 
 where ``\mathcal{G}`` denotes the forward map, ``y \in \mathbb{R}^d`` is the vector of observations
-and ``\eta  \in \mathbb{R}^d`` is additive observational noise. Note that ``p`` is the
-size of the parameter vector ``\theta`` and ``d`` the size of the observation vector ``y``.
+and ``\eta  \in \mathbb{R}^d`` is additive noise encompassing observational and representation (i.e., model) errors. Note that ``p`` is the
```

Comment:
I think the discussion of model error is too deep here. Maybe it would be better to leave this discussion to any examples we have with model error?